safe/definitions/hazard_classifications.py

```diff
@@ -607,12 +867,140 @@
     'exposures': [
         exposure_land_cover,
         exposure_place,
-        exposure_population,
         exposure_road,
         exposure_structure
     ]
 }
 
+# duplicate classes for tsunami hazard based on advice from Pak Hamza and
+# modified for population
+tsunami_hazard_classes = {
+    'key': 'tsunami_hazard_classes',
+    'name': tr('Tsunami classes'),
+    'description': tr(
```

Comment:
Herem instead of having a new block of text `tr()` you can put `tsunami_hazard_class['description']` if it's the same description. You will avoid duplication